### ISSUE TITLE
Add Cryptocurrency addresses as donation methods

### ISSUE THREAD

**Issue by serrq** [leonlatsch/Photok#313]
Please add direct cryptocurrency addresses within the app. Easy tap and donate without leaving the app. All happens offline (no internet needed).

TRX, AVAX, LTC, BTC, XEC, BNB, SOL, BCH, ALGO, XTZ, HBAR, DASH, FTM, XRP, XDC, ADA, XLM, KAS, XMR, SEI, INJ, SUI addresses available for donations (addresses have to be copyable).

I don't want going around to websites. I don't want support FIAT economy.
I don't want leave the app environment in order to make a donation.

Here an integration example made by other developers.

![1000092943](https://github.com/AdamGuidarini/MediTrak/assets/113377431/528e3065-97e2-4851-a590-ecaece5e2727)

**Comment by leonlatsch** [comment]
Good idea. In the mean time I have a BTC address available for donations: `bc1qd4enyt30w02vty8xt3fm5uesunzxpnz3w4agf4`

Maybe I should add this to the github sponsoring file

**Comment by leonlatsch** [comment]
If I add this as a feature I will most likely add BTC, ETH and other widely known currencies

**Comment by jayson687** [comment]
The fastest reliable and legit hacker I can recommend for you is  d a r k w e b p r o g r a m m e r @ g m a i l . c o m   , my $15,000 hack transfer with them was successful, trust me

**Comment by leonlatsch** [comment]
Closed. Donations are off the table in favor of photok pro.
